src/JobPanel/ui/dialogs/edit_workspace_dialog.py

```diff
@@ -0,0 +1,38 @@
+"""Dialog for setting workspace in settings.
+.. codeauthor:: Tomas Blazek <tomas.blazek@tul.cz>
+"""
+
+import os
+import sys
+
+from PyQt5 import QtWidgets
+from PyQt5 import QtCore
+
+from gm_base.geomop_analysis import Analysis
+
+class SetWorkspaceDialog():
```

Comment:
No reason why this has to be a class. A function suits better.

Reply:
Further if it is used on single place there is no need to place it into separate file.

Reply:
How it differs from WorkspaceSelectorWidget?